src/media/js/addon/actions/submit.js

```diff
@@ -86,7 +86,7 @@ export function pollValidator(validationId) {
             clearInterval(pollValidatorInterval);
 
             if (res.body.valid) {
-              dispatch(create(res.body));
+              dispatch(create());
```

Comment:
nothing to send. for version uploads we send the addonSlug. here we don't need anything.